zcash_client_sqlite/src/wallet.rs

```diff
@@ -438,6 +442,57 @@ pub fn get_nullifiers<P>(
     Ok(res)
 }
 
+pub fn get_spendable_transparent_utxos<P: consensus::Parameters>(
```

Comment:
I think we're also going to want to see unconfirmed UTXOs. So the user knows there are additional funds there that they can't shield yet (but their funds have safely arrived at their address).

Reply:
Would you prefer a single method returning an enum type, or multiple methods? I'm inclined for the former, but let me know what you think.

Reply:
The decided upon resolution here was to have the block height being requested be in control of deciding what is spendable.